example/src/App.tsx

```diff
@@ -69,6 +71,7 @@ const ListItem = React.memo(
     return (
       <View key={item.path} style={[styles.listItemContainer]}>
         <View style={styles.listItemWidth}>
+          {isLoading && (<Text style={styles.loadingText}>Loading....</Text>)}
```

Comment:
Can we improve loading text showing?
Can we add activity indicator or text loading in waveform view in example only?

Reply:
Done, Please Check